Marian Zhyganov/Homework5/Homework5/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+Console.WriteLine("Task 1");
+
+var firstNum = 0;
+var secondNum = 0;
+string stop;
+do
```

Comment:
We already learned methods, so better to extract business logic from the main method